AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participants-list/participants-list.component.html

```diff
@@ -33,8 +33,11 @@ <h3 class="govuk-heading-m">{{isEditMode ? 'Participants' : 'Participants added'
         <div class="govuk-grid-row vh-row">
           <div class="govuk-grid-column-two-thirds vhtable-header">
             {{pat.title}} {{pat.first_name}} {{pat.last_name}}
-            <div>
-              {{pat.hearing_role_name}}<span *ngIf="pat.representee">, representing</span>
+            <div *ngIf="!pat.isRepresenting">
```

Comment:
an ngTemplate using *ngifelse would be better suited here

Reply:
Done